apps/web/pages/[user]/[type].tsx

```diff
@@ -43,6 +44,7 @@ export default function Type({
         hideBranding={isBrandingHidden}
         isSEOIndexable={isSEOIndexable ?? true}
         entity={entity}
+        bookingData={booking}
```

Comment:
Here bookingData was required since we were displaying in page title "Reschedule" with just rescheduleUid param